R/count_teleconnections.R

```diff
@@ -73,7 +81,8 @@ count_watershed_teleconnections <- function(data_dir,
                  n_hydro = 0,
                  n_thermal = 0,
                  n_landclasses = 0,
-                 n_cropcover = 0)
+                 n_cropcover = 0,
+                 n_floodcontrol_dams = 0)
```

Comment:
Please keep these headers with only one "_" (makes them easier to work with later if anyone wants to easily drop the "n", for example). So something like `n_floodcontroldams`